packages/example/src/gen/addressbook_pb.ts

```diff
@@ -15,10 +15,9 @@
 // @generated by protoc-gen-es v0.0.8 with parameter "target=ts"
 // @generated from file addressbook.proto (package example, syntax proto3)
 /* eslint-disable */
-/* @ts-nocheck */
```

Comment:
This is wrong, there should be a `@ts-nocheck` unless its turned off with `opt: ts_nocheck=false`.

The default was flipped. Pushed a fix.